cpp/include/cpu_operations.h

```diff
@@ -155,7 +154,20 @@ class CpuOperations {
       return a.inverse();
     }
   }
-  static Matrix<T> Norm(const int &p = 2, const int &axis = 0);
+  static Vector<T> Norm(const Matrix<T> a, const int &p = 2, const int &axis = 0) {
+    int num_rows = a.rows();
+    int num_cols = a.cols();
+    float nval = 0;
+    Vector<T> norm(num_cols);
+    for(int j = 0; j < num_cols; j++) {
+	    for(int i = 0; i < num_rows; i++)
+        nval += pow(a(i,j), 2);
```

Comment:
when p equals to numbers that are other than 2, and axis is not 0, you need to write the code that deal with those situations.